pkg/users/account-create-dialog.js

```diff
@@ -216,7 +216,7 @@ export function account_create_dialog(accounts) {
 
         return password_quality(password, force)
                 .catch(ex => {
-                    errs.password = (ex.message || ex.toString()).replace("\n", " ");
+                    errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17916-20221116-081027-9b3960eb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)